.github/workflows/release.yml

```diff
@@ -361,20 +360,36 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           HOLOCHAIN_TARGET_BRANCH: ${{ needs.vars.outputs.holochain_target_branch }}
           RELEASE_BRANCH: ${{ needs.prepare.outputs.release_branch }}
-          TAG: ${{ needs.prepare.outputs.tag }}
+          LATEST_HOLOCHAIN_TAG: ${{ needs.prepare.outputs.latest_holochain_tag }}
+          LATEST_HOLOCHAIN_VERSION: ${{ needs.prepare.outputs.latest_holochain_version }}
         run: |
-          set -eu
+          set -eux
           cd "${HOLOCHAIN_REPO}"
 
           # a positive condition means the current holochain version has already been released, hence this release doesn't contain holochain
-          if gh release view ${TAG}; then
+          if gh release view ${LATEST_HOLOCHAIN_TAG}; then
             export RELEASE_TAG=${RELEASE_BRANCH}
             export IS_HOLOCHAIN_RELEASE="false"
           else
-            export RELEASE_TAG=${TAG}
+            export RELEASE_TAG=${LATEST_HOLOCHAIN_TAG}
             export IS_HOLOCHAIN_RELEASE="true"
           fi
 
+          # this configuers a single, hardcoded branch to publish "latest" release's
```

Comment:
```suggestion
          # this configures a single, hardcoded branch to publish "latest" releases
```

Reply:
Is this a preparatory step for making alpha releases?

Reply:
yes, so that it doesn't appear as the most-recent release on the github page